data/features/chatGPT/chatGPT.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+
+chatGPT()
+{
+  if [ ! -f "€{BIN_FOLDER}/chatGPT/.env" ]; then
+    echo "You need to enter a valid API key in the file €{BIN_FOLDER}/chatGPT/.env"
+    echo "Go to https://beta.openai.com/account/api-keys to get yours."
+    echo "Edit or create the .env file and write inside the file:"
+    echo "OPENAI_API_KEY=your_open_ai_api_key_code"
+  else
+    if [ ! "$1" ]; then
```

Comment:
what is the condition that you are checking in the if? [ ! "string" ]   ??

Rewrite it with the evaluations that we use in the customizer, this is not how we check the number of args